lib/login.js

```diff
@@ -55,7 +68,7 @@ module.exports = {
             }
         }, function (err, data) {
             if (err) {
-                return callback(err);
+                throw new Error(err);
```

Comment:
Just a precaution to make sure never ever do I write to NPMRC unless absolutely sure.